GraphQL/GraphQL.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net7.0;net8.0</TargetFrameworks>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <PackageId>AElf.GraphQL</PackageId>
+        <Version>1.0.7</Version>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="Volo.Abp.SettingManagement.Application.Contracts" Version="7.0.0" />
```

Comment:
By default, we are using ABP 8.2.0. Check if it corresponds to the package. Where do we require this package?